test-case/test-speaker.sh

```diff
@@ -41,6 +41,9 @@ do
     dev=$(func_pipeline_parse_value $idx dev)
     snd=$(func_pipeline_parse_value $idx snd)
 
+    type=$(func_pipeline_parse_value $idx type)
+    [ "$type" = "playback" ] || continue
```

Comment:
I just realized `type` is pretty bad variable name because it's also a command name. Well, it seems to have worked so far...

Reply:
we also have other scripts use "type" as variable name, we can do clean up later, I am not going to do any change in this PR.